packages/data-service/src/end-of-life-server.ts

```diff
@@ -0,0 +1,61 @@
+import { createLogger } from '@mongodb-js/compass-logging';
+const { mongoLogId, log } = createLogger('END-OF-LIFE-SERVER');
+
+const FALLBACK_END_OF_LIFE_SERVER_VERSION = '4.4';
+const { HADRON_AUTO_UPDATE_ENDPOINT, HADRON_AUTO_UPDATE_ENDPOINT_OVERRIDE } =
+  process.env;
```

Comment:
I wonder if there's a better place to put these. Some package intended to share constants across the main and renderer processes.